pages/item/_itemId/index.vue

```diff
@@ -78,24 +78,39 @@ export default Vue.extend({
         return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Backdrop`;
       }
     },
+    getEndsAtTime(ticks: number): string {
+      const ms = ticks / 10000;
+      const endTimeLong = new Date(Date.now() + ms);
+      const endTimeShort = endTimeLong.toLocaleString('en-US', {
+        hour: 'numeric',
+        minute: 'numeric'
+      });
+
+      return `${this.$t('endsAt')}: ${endTimeShort}`;
+    },
     ticksToTime(ticks: number) {
```

Comment:
Hm, I don't think we should ever convert anything to a time without going through a Date object (Or moment.js or date-fns).

It makes it harder to localize and perform operations on.

Reply:
We can refactor it away in a further PR, since I already let it through, but it's definitely something we should change. Adding a `TODO: Use a Date object` would probably be nice to keep track of it.

Reply:
That make sense. I'm not really sure how time is written in many other languages. I've added the todo.